src/schema/search/index.ts

```diff
@@ -54,13 +54,14 @@ export const searchArgs = pageable({
   },
 })
 
-const fetch = (
-  searchResultItem,
-  { artistLoader, artworkLoader }: ResolverContext
-) => {
+const fetch = (searchResultItem, context: ResolverContext) => {
   const loaderMapping = {
-    Artist: artistLoader,
-    Artwork: artworkLoader,
+    Artist: context.artistLoader,
+    Artwork: context.artworkLoader,
+    Article: context.articleLoader,
+    Fair: context.fairLoader,
+    Gene: context.geneLoader,
+    Profile: context.profileLoader,
```

Comment:
I'm not sure what to do about this mismatch between Gallery / Institution and Profile. Should we make new types that derive from Profile so we can write `... on Gallery` and `... on Institution` fragment queries? Or should we retaining the `Profile` searchableType so can use `... on Profile` in queries?

Reply:
One question- do we need to be able to specify a more specific fragment for galleries and institutions? Or can we use the JSON in the searchable response itself? I think we might be able to just use the searchable response itself (since that's what powers the existing search results page).

Otherwise, I would vote for keeping it `... on Profile`, since that's really what they _are_ (and all searchable JSON will indicate they are profiles, just with specific owner types). The gallery/institution is to be able to search for them separately (and in the Gallery tab, present it separately), but they're all the same thing.

Reply:
Maybe we can introduce a new attribute of `displayType` if we still need that translation to happen here. Leave `searchableType` as "Profile" and match the existing type in Metaphysics. What do you think?